yarn.lock

```diff
@@ -12615,21 +12615,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@rjsf/utils@npm:5.17.1":
```

Comment:
Based on your changes there should be no changes to this file, can you please revert?

Reply:
Removed it from the commit.

But I see that the file is being changed whenever I run yarn install, not sure why though.